db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+Book.delete_all
```

Comment:
Nothing to change here, but you should take a look at `destroy` vs `delete` behavior with active record, especially as it relates to associations with a `:dependent` option set. Just something to be aware of. Here's a helpful stack overflow post as a place to start: http://stackoverflow.com/questions/6698207/delete-all-vs-destroy-all